mitodl/open-discussions
# add rss_url to podcast etl

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3091

#### What's this PR do?
Adds rss_url to podcast etl and serializer

#### How should this be manually tested?
Run `backpopulate_podcast_data` management command and then go to `/api/v0/podcasts`. The rss url should be populated
